Documentation/packaging-installation.md

```diff
@@ -0,0 +1,82 @@
+
+## Installing the SDK globally
+
+If you want to install the SDK globally (and not per user), here are some suggestions.
+
+1. Extract the tarball to a distribution-appropriate location such as `/usr/local/lib64/dotnet/`, `/usr/local/lib/dotnet/`, or `/usr/local/lib/x86_64-linux-gnu/dotnet/`.
+
+2. Create a symlink from `/usr/local/bin/dotnet` (or equivalent location available in `$PATH`) to the `dotnet` binary in SDK that you installed in the previous step. Foe example, `/usr/local/bin/dotnet` -> `/usr/local/lib64/dotnet/dotnet`. This lets users simply run `dotnet` and have it work.
+
+3. Create an `/etc/dotnet/install_location` file and add the path of the SDK directory in there (such as `/usr/local/lib64/dotnet`). This file is used by [.NET Core to find the SDK/Runtime location](https://github.com/dotnet/designs/blob/master/accepted/2020/install-locations.md).
+
+4. Define `DOTNET_ROOT` and update `PATH` by saving the following as `/etc/profile.d/dotnet-local.sh` (or equivalent)
+
+   ```bash
+   # Set location for AppHost lookup
+   [ -z "$DOTNET_ROOT" ] && export DOTNET_ROOT=/usr/local/lib64/dotnet
+
+   # Add dotnet tools directory to PATH
+   DOTNET_TOOLS_PATH="$HOME/.dotnet/tools"
+   case "$PATH" in
+       *"$DOTNET_TOOLS_PATH"* ) true ;;
+       * ) PATH="$PATH:$DOTNET_TOOLS_PATH" ;;
+   esac
+   ```
+
+   Make sure to adjust the paths to match what you used on your system. This allows apphost-lookup to work via `DOTNET_ROOT` and allows users to easily use dotnet tools directly after a `dotnet tool install`. This snippet should work on `sh` (including `dash`) and `bash`. You may need to adapt it for other shells.
+
+## Creating a Linux distribution package
+
+If you want to create a Linux distribution package (`rpm`, `deb`) out of the source-build SDK here are some suggestions.
+
+See [.NET Core distribution packaging](https://docs.microsoft.com/en-us/dotnet/core/distribution-packaging) for information on suggested packages, subpackages, name and contents.
+
+This is the minimal amount of content you need to package up:
+
+1. Extract the tarball to the distribution appropriate location such as `/usr/lib64/dotnet/`, `/usr/lib/dotnet/`, or `/usr/lib/x86_64-linux-gnu/dotnet/`.
+
+2. Create a symlink from `/usr/bin/dotnet` (or equivalent) to the `dotnet` binary in SDK that you installed in the previous step. Foe example, `/usr/bin/dotnet` -> `/usr/lib64/dotnet/dotnet`. This lets users simply run `dotnet` and have it work.
+
+3. Create an `/etc/dotnet/install_location` file and add the path of the SDK directory in there (such as `/usr/lib64/dotnet`). This file is used by [.NET Core to find the SDK/Runtime location](https://github.com/dotnet/designs/blob/master/accepted/2020/install-locations.md).
```

Comment:
```suggestion
3. Create an `/etc/dotnet/install_location` file with a single line of content, the path of the SDK directory (such as `/usr/lib64/dotnet`). This file is used by [.NET Core to find the SDK/Runtime location](https://github.com/dotnet/designs/blob/master/accepted/2020/install-locations.md).
```

Reply:
Thanks for raising this. I have added an "example" file to hopefully make it clearer to the reader.

Reply:
I like it, thanks. 😄